File: SDKs/google.home/2.0/typescript/src/sdk/models/operations/seteurekainfo.ts
import { SpeakeasyMetadata, SpeakeasyBase } from "../../../internal/utils";
import * as shared from "../shared";



export class SetEurekaInfoRequest extends SpeakeasyBase {
  @SpeakeasyMetadata({ data: "request, media_type=application/json" })
  request: shared.SetEurekaInfoRequest;
}


export class SetEurekaInfoResponse extends SpeakeasyBase {
  @SpeakeasyMetadata()
  contentType: string;

  @SpeakeasyMetadata()
  setEurekaInfo200TextPlainObject?: string;

  @SpeakeasyMetadata()
  statusCode: number;
}

File: SDKs/google.home/2.0/python/src/sdk/models/operations/seteurekainfo.py
import dataclasses
from typing import Optional
from ..shared import seteurekainforequest as shared_seteurekainforequest


@dataclasses.dataclass
class SetEurekaInfoRequest:
    request: shared_seteurekainforequest.SetEurekaInfoRequest = dataclasses.field(metadata={'request': { 'media_type': 'application/json' }})
    

@dataclasses.dataclass
class SetEurekaInfoResponse:
    content_type: str = dataclasses.field()
    status_code: int = dataclasses.field()
    set_eureka_info_200_text_plain_object: Optional[str] = dataclasses.field(default=None)
    
